.test-infra/jenkins/job_LoadTests_Combine_Python.groovy

```diff
@@ -0,0 +1,165 @@
+                        input_options       : '\'{"num_records": 20000000,' +
+                                '"key_size": 10,' +
+                                '"value_size": 90,' +
+                                '"bundle_size_distribution_type": "const",' +
+                                '"bundle_size_distribution_param": 1,' +
+                                '"force_initial_num_bundles": 1}\'',
+                        max_num_workers     : 5,
+                        num_workers         : 5,
+                        top_count           : 20,
+                ]
+        ],
+        [
+                title        : 'Combine Python Load test: 2GB 100 kilobyte records',
+                itClass      : 'apache_beam.testing.load_tests.combine_test:CombineTest.testCombineGlobally',
+                runner       : CommonTestProperties.Runner.DATAFLOW,
+                sdk          : CommonTestProperties.SDK.PYTHON,
+                jobProperties: [
+                        job_name            : 'load-tests-python-dataflow-batch-combine-3-' + now,
+                        project             : 'apache-beam-testing',
+                        temp_location       : 'gs://temp-storage-for-perf-tests/smoketests',
+                        publish_to_big_query: true,
+                        metrics_dataset     : 'load_test',
+                        metrics_table       : 'python_dataflow_batch_combine_3',
+                        input_options       : '\'{"num_records": 2000,' +
+                                '"key_size": 100000,' +
+                                '"value_size": 900000,' +
+                                '"bundle_size_distribution_type": "const",' +
+                                '"bundle_size_distribution_param": 1,' +
+                                '"force_initial_num_bundles": 1}\'',
+                        max_num_workers     : 5,
+                        num_workers         : 5,
+                        top_count           : 20,
+                ]
+        ],
+        [
+                title        : 'Combine Python Load test: 2GB Fanout 4',
+                itClass      : 'apache_beam.testing.load_tests.combine_test:CombineTest.testCombineGlobally',
+                runner       : CommonTestProperties.Runner.DATAFLOW,
+                sdk          : CommonTestProperties.SDK.PYTHON,
+                jobProperties: [
+                        job_name            : 'load-tests-python-dataflow-batch-combine-4-' + now,
+                        project             : 'apache-beam-testing',
+                        temp_location       : 'gs://temp-storage-for-perf-tests/smoketests',
+                        publish_to_big_query: true,
+                        metrics_dataset     : 'load_test',
+                        metrics_table       : 'python_dataflow_batch_combine_4',
+                        input_options       : '\'{"num_records": 20000000,' +
```

Comment:
In proposal there is `20000000/4 = 5000000`

Reply:
Oh, that's right. I didn't notice.